### PR TITLE
Support new fluenceJS version in `run` command. Print results of a function

### PR DESCRIPTION
**Pull request by DieMyst**

### CODE REVIEW

File: cli/src/main/scala/aqua/run/RunOpts.scala [hunk 1]
```diff
@@ -25,15 +28,42 @@ object RunOpts {
         "/dns4/kras-00.fluence.dev/tcp/19001/wss/p2p/12D3KooWR4cv1a8tv7pps4HH6wePNaK6gf1Hww5wcCMzeWxyNw51"
```
Comment (alari) [inline cli/src/main/scala/aqua/run/RunOpts.scala:28]: Can you avoid hardcoding that?

File: cli/src/main/scala/aqua/run/RunOpts.scala [hunk 2]
```diff
@@ -25,15 +28,42 @@ object RunOpts {
...
+      .mapValidated { str =>
+        CallArrowExpr.funcOnly.parseAll(str) match {
+          case Right(f) =>
+            val expr = f.mapK(spanToId)
+            val hasVars = expr.args.exists {
+              case VarLambda(_, _) => true
+              case _ => false
+            }
+            if (hasVars) {
+              Validated.invalidNel("Function must have only literals as variables")
```
Comment (alari) [inline cli/src/main/scala/aqua/run/RunOpts.scala:50]: Could it contain constants from the compiled file?

**Review by alari — APPROVED** [review]

File: cli/src/main/scala/aqua/run/RunOpts.scala (hunk 2 above)
Comment (alari) [reply cli/src/main/scala/aqua/run/RunOpts.scala:50]: `Function can have only literal arguments, no variables or constants allowed at the moment`

File: cli/.js/src/main/scala/aqua/RunCommand.scala [hunk 3]
```diff
@@ -38,34 +44,80 @@ object RunCommand extends Logging {
...
         .toFuture
       peer = Fluence.getPeer()
+      promise = Promise.apply[Unit]()
       _ = CallJsFunction.registerUnitService(
         peer,
-        "console",
-        "print",
-        args => println("print: " + args)
+        config.consoleServiceId,
+        config.printFunctionName,
```
Comment (folex) [inline cli/.js/src/main/scala/aqua/RunCommand.scala:59]: This seems to be not just `console` service, but a completion signal. Maybe call it something like `wrapCallbackSrv` with value like `--after-callback-srv-service--` and function name `print-and-stop`

File: cli/.js/src/main/scala/aqua/RunCommand.scala [hunk 4]
```diff
@@ -38,34 +44,80 @@ object RunCommand extends Logging {
...
+        args => {
+          // if an input function returns a result, our success will be after it is printed
+          // otherwise finish after JS SDK will finish sending a request
+          println("result: " + args)
```
Comment (folex) [inline cli/.js/src/main/scala/aqua/RunCommand.scala:63]: I think there's no need for `result:` prefix, really. But I'm not sure. Do you have a screenshot of execution?

**Pull request merged**